src/microbe_stage/AvailableUpgrade.cs

```diff
@@ -7,7 +7,15 @@
 /// </summary>
 public class AvailableUpgrade : IRegistryType
 {
+    private LoadedSceneWithModelInfo loadedSceneData;
+
 #pragma warning disable 169,649 // Used through reflection
+    /// <summary>
+    ///   A path to a scene to display this organelle with. If empty won't have a display model.
```

Comment:
I don't think this comment matches what the variable actually does in an upgrade.